apps/frontend/src/pages/notifications/libs/constants/query-parameter-name.constant.ts

```diff
@@ -0,0 +1,3 @@
+const QUERY_PARAMETER_NAME = "type";
```

Comment:
```suggestion
const NOTIFICATION_TYPE_QUERY_PARAMETER_NAME = "type";
```
We can have lots of query parameters